File: interface/src/main/java/com/kgc/pinyougou/service/GoodsService.java
package com.kgc.pinyougou.service;

import com.kgc.pinyougou.domain.Goods;
import com.kgc.pinyougou.utils.PageResult;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface GoodsService {

	/**
	 * @Author Admin
	 * @description:  //TODO
	 * @param:
	 * @param pageNum   分页页数
	 * @param pageSize  分页记录数
	 * @return: com.kgc.pinyougou.utils.PageResult
	 * @time: 2020/4/25 22:11
	 */
	PageResult findAllGoods(Integer pageNum, Integer pageSize, Goods goods);

	Integer insertGoods(Goods goods);

	Goods findOne(Long id);

	Integer updateGoods(Goods goods);

	Integer deleteGoods(Long [] ids);
}

File: pinyougou-manager-web/src/main/resources/static/js/service/goodsService.js
//服务层
app.service('goodsService',function($http){

    this.findAllGoods=function (page,size,searchEntity) {
        return $http.post("../goods/findAllGoods?page="+page+"&size="+size,searchEntity)
    }

    this.findOne=function (id) {
        return $http.get("../goods/findOne?id="+id)
    }

    this.del=function (ids) {
        return $http.post("../goods/delGoods",ids)
    }

    this.add=function (searchEntity) {
        return $http.post("../goods/addGoods",searchEntity)
    }

    this.update=function (searchEntity) {
        return $http.post("../goods/updateGoods",searchEntity)
    }
});
